src/foundry/install.sh

```diff
@@ -26,11 +26,12 @@ echo "Installing 'foundryup'"
 curl -L https://foundry.paradigm.xyz \
 	| FOUNDRY_DIR="${ROOT}" PATH="${BIN}:${PATH}" bash
 
+VAR_AT='$@'
 cat > /usr/local/bin/foundryup << EOF
 #!/bin/sh
 export FOUNDRY_DIR="${ROOT}"
 export PATH="${BIN}:\${PATH}"
-/opt/foundry/bin/foundryup
+/opt/foundry/bin/foundryup "${VAR_AT}"
```

Comment:
Out of curiosity, why not just:

```suggestion
/opt/foundry/bin/foundryup "\$@"
```